src/WebJobs.Script.WebHost/WebScriptHostManager.cs

```diff
@@ -433,5 +432,11 @@ internal void InitializeHttpFunctions(IEnumerable<FunctionDescriptor> functions)
                 }
             }
         }
+
+        public override void Shutdown()
+        {
+            Stop();
+            HostingEnvironment.InitiateShutdown();
```

Comment:
How are we logging these shutdown events? Seems like we should have log entries similar to our other Info level start/stop events.